src/microbe_stage/StrictProcessDisplayInfoEquality.cs

```diff
@@ -33,6 +33,9 @@ public bool Equals(StrictProcessDisplayInfoEquality? other)
         var our = DisplayInfo;
         var theirs = other.DisplayInfo;
 
+        if (our.Enabled != theirs.Enabled)
```

Comment:
The whole process equality comparisons is a bit of a mess and changing any of the code here can cause bugs in a bunch of places. I think the enabled state probably should not be compared here as having a process speed display show the same item twice but another is disabled while the other is enabled doesn't seem like a good idea to me.